tests/channel_test.c

```diff
@@ -645,11 +645,19 @@ static int s_test_channel_connect_some_hosts_timeout(struct aws_allocator *alloc
 
     aws_io_library_init(allocator);
 
-    struct aws_event_loop_group event_loop_group;
-    ASSERT_SUCCESS(aws_event_loop_group_default_init(&event_loop_group, allocator, 1));
-
     struct aws_mutex mutex = AWS_MUTEX_INIT;
 
+    struct channel_connect_test_args callback_data = {
+        .mutex = &mutex,
+        .cv = AWS_CONDITION_VARIABLE_INIT,
+        .error_code = 0,
+        .channel = NULL,
+        .setup = false,
+        .shutdown = false,
+    };
+
+    struct aws_event_loop_group *event_loop_group = aws_event_loop_group_new_default(allocator, 1, NULL);
+
```

Comment:
`ASSERT_NOT_NULL(event_loop_group)`

This applies to pretty much every test?
or maybe it doesn't matter because everything would just explode?

Reply:
Yeah, I've never understood asserting on creation in a test.  A crashed test is a failed test.